src/main/scala/is/hail/variant/VariantSampleMatrix.scala

```diff
@@ -843,14 +817,6 @@ class VariantSampleMatrix[RPK, RK, T >: Null](val hc: HailContext, val metadata:
     })
   }
 
-  def deleteGlobal(args: String*): (Type, Deleter) = deleteGlobal(args.toList)
-
-  def deleteGlobal(path: List[String]): (Type, Deleter) = globalSignature.delete(path)
-
-  def deleteSA(args: String*): (Type, Deleter) = deleteSA(args.toList)
-
-  def deleteSA(path: List[String]): (Type, Deleter) = saSignature.delete(path)
-
```

Comment:
I'm a big fan of code removal, but a lot of this seems independent of the rest of the PR.